DirectEmployers/MyJobs
# [PD-1555] First cut at dynamic reports api and throwaway ui.

This provides a basic UI showing some API interaction from a real-ish page.

~~I don't yet have a strategy for mapping users to usertypes.~~ There are two usertypes and there is a single function and tests for making the determination.

To test this, load the provided fixture:

1. python manage.py loaddata myreports class_and_pres.json
2. Then navigate to https://qc.secure.my.jobs/reports/view/dynamicoverview
3. There should be buttons. Click them to see the flow.
4. Also watch network activity in the browser's debugger to see the nature of the api as it stands.

Edit: Now with working user types.
